pkg/sql/insert.go

```diff
@@ -158,7 +160,10 @@ func (r *regionLocalInfoType) checkHomeRegion(row tree.Datums) error {
 	return nil
 }
 
-func (r *insertRun) initRowContainer(params runParams, columns colinfo.ResultColumns) {
+func (r *insertRun) init(params runParams, columns colinfo.ResultColumns) {
+	if ots := params.extendedEvalCtx.SessionData().OriginTimestampForLogicalDataReplication; ots.IsSet() {
```

Comment:
uber nit: in other `*run` functions, we set the helper after the  `r.rowsNeeded` check. I don't think it matters.

Reply:
I don't think it matters because the sql writer plans all end up requiring rows, but I think it is slightly more correct to init the cput helper before checking rowsNeeded.